packages/realm-react/README.md

```diff
@@ -119,7 +119,7 @@ For a full fledged example, check out [our templates](https://github.com/realm/r
 ## Realm Hooks
 
 ### useRealm
-Returns the instance of the [`Realm`](https://www.mongodb.com/docs/realm-sdks/js/latest/Realm.html) configured by `createRealmContext` and the `RealmProvider`.  The following is an example of how to use this Hook to make a write transaction callback for a component.
+Returns the instance of the [`Realm`](https://www.mongodb.com/docs/realm-sdks/js/latest/classes/Realm.html) configured by `createRealmContext` and the `RealmProvider`.  The following is an example of how to use this Hook to make a write transaction callback for a component.
```

Comment:
This is giving me a 403 Forbidden.

Reply:
Yes, something is wrong in our API documentation. Currently https://www.mongodb.com/docs/realm-sdks/js/latest/classes/export_-1.html is generated. This PR is assuming we fix the API documentation.

Reply:
Our [reference docs](https://www.mongodb.com/docs/realm-sdks/js/latest/modules.html#Realm) (especially for the `Realm` class) actually seem to have been affected by the export refactor 🤔

Reply:
I'll try to fix it in a separate PR

Reply:
> the export refactor

What PR are you referring to specifically?

Reply:
Likely the [Rollup one](https://github.com/realm/realm-js/pull/6492), where we started using `export import`.

Reply:
Yeah - I don't remember ever publishing docs from that change.

Reply:
The published docs are 12.6.1, and it seems that Rollup was removed in that release.

Reply:
The v12.7.0 API docs looks fine